pkg/costmodel/allocation_helpers.go

```diff
@@ -1818,7 +1821,13 @@ func applyPVBytes(pvMap map[pvKey]*pv, resPVBytes []*prom.QueryResult) {
 			continue
 		}
 
-		pvMap[key].Bytes = res.Values[0].Value
+		pvBytesUsed := res.Values[0].Value
+		if pvBytesUsed >= PV_USAGE_SANITY_LIMIT {
+			pvMap[key].Bytes = pvBytesUsed
+		} else {
+			pvMap[key].Bytes = 0
+			log.Infof("[WARNING] PV usage exceeds sanity limit, clamping to zero")
+		}
```

Comment:
This looks backwards to me -- can we double-check? If the measured bytes exceed the limit, we want to clamp; if they do not exceed, we want to use the measured value, right?

Reply:
Yep, I accidentally swapped that, it's fixed now